addons/spreadsheet/static/tests/helpers/pivot.js

```diff
@@ -65,34 +98,19 @@ export async function insertPivotInSpreadsheet(model, pivotId, params) {
         pivotId,
         pivot,
     });
-    const ds = model.getters.getPivot(pivotId);
-    if (!(ds instanceof OdooPivot)) {
-        throw new Error("The pivot data source is not an OdooPivot");
+    if (params.mode === "static") {
```

Comment:
```suggestion
    if (params.pivotType === "static") {
```
maybe ? Something clearer than mode.

Reply:
should also add mode to the docstring's typing

Reply:
Updated :)